Mathlib/RingTheory/MvPolynomial/Basic.lean

```diff
@@ -148,6 +149,19 @@ theorem linearIndependent_X : LinearIndependent R (X : σ → MvPolynomial σ R)
 set_option linter.uppercaseLean3 false in
 #align mv_polynomial.linear_independent_X MvPolynomial.linearIndependent_X
 
+instance {R σ : Type*} [CommSemiring R] [Finite σ] (N : ℕ) :
+    Module.Finite R (restrictTotalDegree σ R N) :=
```

Comment:
Can you add `Module.Finite R (restrictDegree σ R N)` too? It's the same proof but without the `have : Finite {s : σ →₀ ℕ | s.sum (fun _ e ↦ e) ≤ N}` part.